common_logging/src/main/java/uk/gov/dhsc/htbhf/logging/event/ApplicationStartedEvent.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.dhsc.htbhf.logging.event;
+
+import lombok.Builder;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+/**
+ * An {@link Event} marking the start of an application instance.
+ */
+public class ApplicationStartedEvent extends Event {
+
+    @Builder
+    public ApplicationStartedEvent(String applicationVersion, String applicationId, String instanceIndex) {
+        super(CommonEventType.APPLICATION_STARTED,
+                LocalDateTime.now(),
+                constructMetadata(applicationVersion, applicationId, instanceIndex));
+    }
+
+    private static Map<String, Object> constructMetadata(String applicationVersion, String applicationId, String instanceIndex) {
+        return Map.of(
+                "applicationVersion", applicationVersion,
```

Comment:
Should these be constants, or do the same as the claimant-service and have and enum with these keys in?

Reply:
I left them as sytrings here, as this is the only place they're mentioned (other than in the test). As long as clients use the `ApplicationStartedEvent` class there's no need to re-use the constants. I could move them into constants on the class though.